CHANGELOG.md

```diff
@@ -34,6 +36,18 @@ librdkafka v2.4.0 is a feature release:
    depending on the application logic (#4583).
 
 
+## Early Access
```

Comment:
I think its better to put KIP-848 related section in itself so that we emphasize that this is a major feature we care about.